src/manager.js

```diff
@@ -0,0 +1,40 @@
+class Manager {
+  constructor() {
+    this.cohortList = []
+  }
+
+  createCohortName(cohortName) {
+    const cohort = {
+      name: cohortName,
+      studentList: []
+    }
+    this.cohortList.push(cohort)
+    return this.cohortList
+  }
+
+  searchCohortName(cohortName) {
+    const newArray = []
```

Comment:
on this one, you created a newArray, and you return the newArray...but when you search for a cohort by name in this case you would only return 1 cohort - so you would not need to create a newArray - just return the found cohort by itself.